elasticsearch-akka/src/test/scala/com/sumologic/elasticsearch/akkahelpers/BulkIndexerActorTest.scala

```diff
@@ -26,17 +26,20 @@ import com.sumologic.elasticsearch.akkahelpers.BulkIndexerActor.{BulkSession, Cr
 import com.sumologic.elasticsearch.restlastic.{RestlasticSearchClient, RestlasticSearchClient6}
 import com.sumologic.elasticsearch.restlastic.RestlasticSearchClient.ReturnTypes.BulkItem
 import com.sumologic.elasticsearch.restlastic.dsl.Dsl._
+import org.junit.runner.RunWith
 import org.mockito.ArgumentMatchers._
 import org.mockito.Matchers._
 import org.mockito.Mockito._
 import org.scalatest._
 import org.scalatest.concurrent.Eventually
 import org.scalatest.mock.MockitoSugar
+import org.scalatestplus.junit.JUnitRunner
 
 import scala.concurrent.duration._
 import scala.concurrent.{Await, Future}
 import scala.concurrent.duration.Duration
 
+@RunWith(classOf[JUnitRunner])
```

Comment:
It'd be great if we didn't need this.  It caused issues in our internal repos where people forgot to add it.